halium-install

```diff
@@ -38,6 +38,35 @@ do_shell()
 	adb shell "$@"
 }
 
+do_chroot()
+{
+	do_shell "chroot /cache/system $@"
+}
+
+check_busybox()
+{
+    export device_architecture=$(do_shell "uname -m")
+    export busybox_exists=$(do_shell "if [ -d /sbin/busybox ]; then echo true; else echo false; fi")
+    mkdir busybox
+
+    if [ $busybox_exists = "true" ]; then
+        echo 'Using already installed busybox'
+    else
+        if [ $device_architecture = x86_64 ]; then
+            wget https://github.com/meefik/busybox/raw/master/app/src/main/assets/intel/static/bin/busybox -O busybox/busybox
```

Comment:
We could actually build our own busybox, or just use a prebuilt static one from busybox.org? I wouldn't necessarily use busybox from a random busybox app.